src/app/body/task-details/activity/activity.component.ts

```diff
@@ -26,26 +25,13 @@ export class ActivityComponent implements OnInit {
 
   @Input('activity') activity: Activity
 
-  public userObservable: Observable<UserAppSetting>;
+  user: User;
 
-  user: UserAppSetting;
-
-  constructor(private userService: UserServiceService) { }
+  constructor(public authService: AuthService) { }
 
   ngOnInit(): void {
-    this.getUserDetail();
+    
   }
 
-  getUserDetail() {
-    if(this.activity.Uid == "defaultUser"){
-      this.user = defaultUser;
```

Comment:
Merge conflict is not resolved properly
getUserDetails is needed hear i guess

Reply:
reverted the changes